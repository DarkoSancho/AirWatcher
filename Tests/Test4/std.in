government official
2
44.0
0.0
01/01/2019 00:00
01/01/2019 23:59