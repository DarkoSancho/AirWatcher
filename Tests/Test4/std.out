==== RESULTS ====

Average air quality in the zone  
Center: 44.0, 0.0 -- Radius: 5 km  
Period: 01/01/2019 00:00 → 01/01/2019 23:59

→ ATMO Index = 3 (Moderate air quality)
